dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/trackedentity/store/mapper/EnrollmentRowCallbackHandler.java

```diff
@@ -108,8 +108,8 @@ private Enrollment getEnrollment( ResultSet rs )
         enrollment.setStatus( EnrollmentStatus.fromStatusString( rs.getString( getColumnName( STATUS ) ) ) );
         enrollment.setEnrollmentDate( rs.getTimestamp( getColumnName( ENROLLMENTDATE ) ) );
         enrollment.setIncidentDate( rs.getTimestamp( getColumnName( INCIDENTDATE ) ) );
-        final String followup = rs.getString( getColumnName( FOLLOWUP ) );
```

Comment:
This returns "t" and Boolean.parseBoolean( "t" ) is false. Changed it to use resultSet.wasNull() to preserve null